zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/SQLPPGenerator.scala

```diff
@@ -264,7 +279,8 @@ class SQLPPGenerator extends AsterixQueryGenerator {
           } else {
             s"select value $sourceVar"
           }
-        ParsedResult(Seq(projectStr, "", "", ""), exprMap)
+        queryBuilder.insert(0, projectStr + "\n")
```

Comment:
Here queryBuilder is used to incrementally build the query. `projectStr` is actually a select statement, and thus has to be appeared in the beginning of the query.